model/key.go

```diff
@@ -60,6 +60,21 @@ type KeyInfo struct {
 	// KMIP Key ID, if the key is already created in KMIP Backend
 	// example: 7110194b-a703-4657-9d7f-3e02b62f2ed8
 	KmipKeyID string `json:"kmip_key_id,omitempty"`
+	// The OCID of the compartment where you want to create the secret.
+	// example: ocid1.test.oc1..<unique_ID>EXAMPLE-compartmentId-Value
+	OciCompartmentId string `json:"oci_compartment_id,omitempty"`
```

Comment:
do we need to return all these fields in response? if not, then we can keep these in separate struct.

Reply:
Yeah, that's what I was thinking too. I just did this for now since the Vault and KMIP stuff was also all mixed in there. If we want to separate them out, then I can do a patch that first pulls out the Vault and KMIP stuff into separate structs, then change this patch to also be in a separate struct.

Reply:
we only have KmipKeyID field for KMIP which is needed in both request and response (none specific to vault), hence kept KmipKeyID in KeyInfo struct which is shared between request and response. If the OCI fields also need to be kept in both request and response, then we can leave them in KeyInfo struct.

Reply:
While separating out the OCI stuff, I was going over the other code. Both `KeyData` and `KmipKeyID` are actually both only used with KeyRequest as well. They are also in the KeyAttributes struct. Is the intent is that KeyAttributes and KeyInfo should roughly match?

If so, one option for the OCI code is to just put `OciSecretId` in KeyInfo and KeyAttributes, as that's the only one needed in both. Then I can put the rest of the OCI fields into their own struct and just include that in the KeyRequest struct